docs/reference/feast-cli-commands.md

```diff
@@ -292,6 +321,47 @@ reader             driver_hourly_stats_fresh   FeatureView      DESCRIBE
 ...
 ```
 
+## Tag
+Update tags for the following objects types:
+- data-source
+- entity
+- feature-service
+- feature-view
+- on-demand-feature-view
+- permission
+- project
+- saved-dataset
+- stream-feature-view
+- validation-reference
+
+```text
+% feast tag --help
+Usage: feast tag [OPTIONS] OBJECT_TYPE NAME TAGS...
+
+  [Experimental] Tag objects
+
+  *  A tag key and value must begin with a letter or number, and may contain
+  letters, numbers, hyphens, dots, and underscores, up to 63 characters each.
+
+  *  Optionally, the key can begin with a DNS subdomain prefix and a single
+  '/', like example.com/my-app. The 'feast.dev/' prefix is reserved.
+
+  Examples:
+
+      # Update data source 'foo' with the tag 'test' and the value 'true'.
+
+      feast tag data-source foo test:true
+
+      # Update feature view 'foo' by removing a tag named 'bar' if it exists.
+      Does not require the --overwrite flag.
+
+      feast tag feature-view foo bar-
+
+Options:
+  --overwrite  If true, allow tags to be overwritten, otherwise reject tag
+               updates that overwrite existing tags.
```

Comment:
Can we add the default value in the description? I presume it is False.

Reply:
hmm... it should be showing already, but `show_default=True` isn't being honored here for some reason. i'll look into it.

Reply:
Can probably just add the default as a comment